examples/control_section.py

```diff
@@ -1,8 +1,9 @@
 import xarray as xr
 import xrviz
-from xrviz.control import Control
+from xrviz.dashboard import Dashboard
 
 ds = xrviz.sample_data.great_lakes
+ds = ds.set_coords(['lat', 'lon'])
```

Comment:
This cannot be something that we require, we need a way to optionally do this within the code.

Reply:
> optionally do this within the code.

Can you please clarify this more on which part of code ? Do we need to do this within the interface since examples are outside the main xrviz module code?

Reply:
Within the xrviz package, because we cannot assume that the user has somehow pre-processed their datasets.

Reply:
And by "optionally", I mean that the user can choose to use alternate coordinates or not - implementing this ability is not optional.

Reply:
Thanks for the clarification. I have added this task to my todo list. It would be done in a separate PR.

Reply:
This Great Lakes dataset highlights that sometimes we will have datasets that don't have the desired coordinates because the providers didn't follow conventions or some other problem.

We want the GUI to display the coordinate variables, as well as listing variables that potentially could be coordinates, and allow the user to select coordinates manually.   Is this your idea, @martindurant?

Reply:
> We want the GUI to display the coordinate variables, as well as listing variables that potentially could be coordinates, and allow the user to select coordinates manually.

Exactly; in this case, nx,ny,time are initially the coordinates available, and we can infer from the structure of the dimensions of each coordinate/variable that we can map nx/ny to lat/lon. The user needs to be able to plot against any combination of nx/ny/time and limited combinations of lat/lon/time. Exactly how to pick, and how to get xarray to tell us what is allowed or possible, I'm not too sure.

Reply:
This task would be done in next PR as discussed in e-meet, after discussion with Pangeo community and with @rsignell-usgs.